charts/argo-cd/Chart.yaml

```diff
@@ -28,3 +28,5 @@ annotations:
   artifacthub.io/changes: |
     - kind: changed
       description: Bump argocd-extension-installer to v0.0.8
```

Comment:
You also need to remove old changelog entry

Reply:
thanks done